src/main/privacy/Resources/modules/account/privacy/modals/terms/components/modal.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import {PropTypes as T} from 'prop-types'
+import omit from 'lodash/omit'
+
+import {trans} from '#/main/app/intl/translation'
+import {Modal} from '#/main/app/overlays/modal/components/modal'
+import {ContentLoader} from '#/main/app/content/components/loader'
+import {ContentHtml} from '#/main/app/content/components/html'
+
+const TermsModal = (props) => 
+  <Modal
+    {...omit(props, 'fetch')}
+    icon="fa fa-fw fa-shield"
+    title={trans('terms_of_service')}
+    bsSize="lg"
+    onEntering={() => {
+      if (!props.loaded) {
+        props.reset(props.termsOfService)
+      }
+    }}
+  >
+    {!props.loaded &&
+        <ContentLoader
+          size="lg"
+          description="Nous chargeons les conditions d'utilisation..."
```

Comment:
Il manque une trad.

Reply:
c'est un copier/coller de ta modal qui se situe dans src/main/app/Resources/modules/modals/terms-of-service/components/modal.jsx

Reply:
je sais, mais qu'importe. Si il y a une trad qui manque on la met, même si c'est pas toi qui l'a fait ;)